src/arch/mips/lib/debug/stack_iter.c

```diff
@@ -0,0 +1,83 @@
+/**
+ * @brief
+ * @author Nastya Nizharadze
+ * @date 30.07.2019
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+#include <hal/context.h>
+#include <util/lang.h>
+#include <util/math.h>
+#include <assert.h>
+#include "stack_iter.h"
+#include "debug/symbol.h"
+
+/* Mips stack:
+ *
+ * low address ^
+ * 
+ *  -------- frame begining
+ * |        | ---		<- new fp + offset
+ * |   ra   |  ^		<- new fp + offset - 4
+ * | old fp |  |		<- new fp + offset - 8
+ * |   ...  |  | offset
+ * |        |  |
+ * |        |  v
+ * |        | ---		<- new fp
+ *  -------- frame ending
+ * 
+ *  */
+
+int offset (void *ra) {
```

Comment:
Is this function used outside of this file? If no, make it `static`, please

Reply:
`offset (`->`offset(`

Reply:
done